test/unit/storage/storage.spec.ts

```diff
@@ -115,17 +115,20 @@ describe('Storage', () => {
   });
 
   describe('Emulator mode', () => {
-    const EMULATOR_HOST = 'http://localhost:9199';
+    const EMULATOR_HOST = 'localhost:9199';
 
     before(() => {
       delete process.env.STORAGE_EMULATOR_HOST;
       process.env.FIREBASE_STORAGE_EMULATOR_HOST = EMULATOR_HOST;
     });
 
     it('sets STORAGE_EMULATOR_HOST if FIREBASE_STORAGE_EMULATOR_HOST is set', () => {
-      new Storage(mockApp);
-      
-      expect(process.env.STORAGE_EMULATOR_HOST).to.equal(EMULATOR_HOST);
+      new Storage(mockApp)
+      expect(process.env.STORAGE_EMULATOR_HOST).to.equal(`http://${EMULATOR_HOST}`);
+    });
+
+    it('throws if FIREBASE_STORAGE_EMULATOR_HOST has a protocol', () => {
```

Comment:
This test is currently passing by accident. There's nowhere in this test that we have set an env variable with the `http://` prefix. If you update the assertion as follows the error surfaces immediately:

```js
expect(() => new Storage(mockApp)).to.throw(
  'FIREBASE_STORAGE_EMULATOR_HOST should not contain a protocol');
```